### PR TITLE
fix: defaultly to play audio

### PR DESCRIPTION
**Pull request by woniuxingdong**
Let user know that video catched do have audio just as they played in the application.

### CODE REVIEW

File: packages/main/src/services/WebviewService.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ export default class WebviewService {
       },
     });
     this.view.setBackgroundColor("#fff");
-    this.view.webContents.setAudioMuted(true);
+    this.view.webContents.setAudioMuted(false);
```
Comment (caorushizi) [inline packages/main/src/services/WebviewService.ts:66]: Can you add an option in the Settings to control the sound playing of the page, the default is false

Comment (woniuxingdong) [reply packages/main/src/services/WebviewService.ts:66]: Doable. Can you tell me why it should be muted by default? It seems strange to me.

**Comment by woniuxingdong** [comment]
Changed to add one setting item to control if the browser need play audio. Here's the screen.
image

**Comment by caorushizi** [comment]
thx

**Pull request merged**